.goreleaser.yml

```diff
@@ -140,3 +142,93 @@ changelog:
+    license: "Apache 2.0"
+    caveats: |
+      ****************************************************************
+      After installation please run the following to finish setup:
+
+        # Default location for the opentelemetry-java-contrib-jmx-metrics.jar
+        sudo cp #{lib}/opentelemetry-java-contrib-jmx-metrics.jar /opt/
+
+      The configuration file that is run by the service is located at #{prefix}/config.yaml.
+
+      If you are configuring a logreceiver the plugin directory is located at #{prefix}/plugins.
+      ****************************************************************
+
+      ****************************************************************
+      Below are services commands to run to manage the collector service.
+      If you wish to run the collector at boot prefix these commands with sudo
+      otherwise the service will run at login.
+
+      To start the collector service run:
+
+        brew services start observiq/observiq-otel-collector/observiq-otel-collector
+
+      To stop the collector service run:
+
+        brew services stop observiq/observiq-otel-collector/observiq-otel-collector
+
+      To restart the collector service run:
+
+        brew services restart observiq/observiq-otel-collector/observiq-otel-collector
+      ****************************************************************
+
+      ****************************************************************
+      To uninstall the collector and its dependencies run the following commands:
+
+          brew services stop observiq/observiq-otel-collector/observiq-otel-collector
+          brew uninstall observiq/observiq-otel-collector/observiq-otel-collector
+          launchctl remove com.observiq.collector
+
+          # If you moved the opentelemetry-java-contrib-jmx-metrics.jar
+          sudo rm /opt/opentelemetry-java-contrib-jmx-metrics.jar
+
+      ****************************************************************
+    install: |
+      bin.install "observiq-otel-collector"
+      prefix.install "LICENSE", "config.yaml"
+      prefix.install Dir["plugins"]
+      lib.install "opentelemetry-java-contrib-jmx-metrics.jar"
+    plist: |
+      <?xml version="1.0" encoding="UTF-8"?>
+      <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+      <plist version="1.0">
+        <dict>
+
+          <key>Label</key>
+          <string>com.observiq.collector</string>
+
+          <key>ProgramArguments</key>
+          <array>
+            <string>#{bin}/observiq-otel-collector</string>
+            <string>--config</string>
+            <string>#{prefix}/config.yaml</string>
+          </array>
+
+          <key>RunAtLoad</key>
+          <true/>
+
+          <key>StandardErrorPath</key>
+          <string>/tmp/observiq-otel-collector.log</string>
+
+          <key>StandardOutPath</key>
+          <string>/tmp/observiq-otel-collector.log</string>
```

Comment:
Yeah they aren't rotated. The best I could find to do rotation was to wrap the core program in a script that will setup log rotation. Since macOS is likely not a major use case of our collector I think solving this problem down the road is fine.